kbfsgit/file_info.go

```diff
@@ -7,41 +7,63 @@ package kbfsgit
 import (
 	"os"
 	"time"
+
+	"github.com/keybase/kbfs/libfs"
+	"github.com/keybase/kbfs/libkbfs"
 )
 
 // FileInfo is a wrapper around libkbfs.EntryInfo that implements the
 // os.FileInfo interface.
 type FileInfo struct {
+	fs       *FS
+	ei       libkbfs.EntryInfo
+	fullpath string
 }
 
 var _ os.FileInfo = (*FileInfo)(nil)
 
 // Name implements the os.FileInfo interface for FileInfo.
 func (fi *FileInfo) Name() string {
-	return ""
+	return fi.fullpath
```

Comment:
Actually, that's the comment for `File.Name`, not `FileInfo.Name`. I think `os.FileInfo` interface still require this to be the base name?